Add infos about Wowza streaming configuration changes to upgrade guide (#2298)
opencast/opencast: docs/guides/admin/docs/upgrade.md

  [this document](https://github.com/elan-ev/opencast-studio/blob/2020-09-14/CONFIGURATION.md).

### Wowza streaming configuration changes

In [pull request #1179](https://github.com/opencast/opencast/pull/1179), the configuration of Wowza servers was made
tenant-specific. Before, you had to specify `etc/org.opencastproject.streaming.url` in `custom.properties`. Starting
with Opencast 9, that value has to be specified as `org.opencastproject.<tenant>.wowza.url` in

> `org.opencastproject.distribution.streaming.wowza.WowzaStreamingDistributionService.cfg`.

Reviewer: Maybe mention the port here as well? Since both URL and port config were moved.

  [this document](https://github.com/elan-ev/opencast-studio/blob/2020-09-14/CONFIGURATION.md).

### Wowza streaming configuration changes

In [pull request #1179](https://github.com/opencast/opencast/pull/1179), the configuration of Wowza servers was made
tenant-specific. Before, you had to specify `etc/org.opencastproject.streaming.url` in `custom.properties`. Starting

> with Opencast 9, that value has to specified as `org.opencastproject.mh_default_org.wowza.url` in

Reviewer: _be_ specified

> with Opencast 9, that value has to specified as `org.opencastproject.mh_default_org.wowza.url` in

Reviewer: Also I'd rather see \<tenant\> in the key instead of "hardcoding" the default tenant. x)

> In [pull request #1179](https://github.com/opencast/opencast/pull/1179), the configuration of Wowza servers was made
> tenant-specific. Before, you had to specify `org.opencastproject.streaming.url` and
> `org.opencastproject.streaming.port` in `etc/custom.properties`. Starting with Opencast 9, those values have to be
> specified as `org.opencastproject.<tenant>.wowza.url` and `org.opencastproject.<tenant>.wowza.port` in
> `etc/org.opencastproject.distribution.streaming.wowza.WowzaStreamingDistributionService.cfg`.

Reviewer 2: You are right, we don't need to mention twice that it is now tenant-specific.
My main intention here is to not be as specific as you are. The configuration file is new and people should look at it and the documentation it contains. Your specific suggested change implies that users should ignore the file but for this key. Also, there is documentation about this in the file and in the docs.
So, I would simply go with:

```suggestion
In [pull request #1179](https://github.com/opencast/opencast/pull/1179), the configuration of Wowza servers was made
tenant-specific and has been moved to `etc/org.opencastproject.distribution.streaming.wowza.WowzaStreamingDistributionService.cfg`.
```

That should give users all information they need, shouldn't it?

Author: Fiiiiiiine. This strange desire to remove everything from docs that not strictly necessary :D 
I pushed. Should be fine now.

Also as part of that pull request, support for RTMP was dropped. This shouldn't be a problem in most cases, however. If
your streaming URL started with `rtmp://`, try changing it to `https://`. In most cases, RTMP wasn't even used before,
despite it being specified in the URL. Instead, the formats/protocols that can be used are specified in

> `org.opencastproject.wowza.formats` at the top of the same file.

Reviewer: Maybe this could be a bit more concise?

Author: Mhhhh what about this?

> Additionally, support for RTMP was dropped. This shouldn't be a problem in most cases, as RTMP usually wasn't used before, despite being specified in the URL. If your streaming URL started with `rtmp://`, try changing it to `https://`. Streaming formats/protocols can be specified in `org.opencastproject.wowza.formats` at the top of the same file.

Reviewer: I'd scratch the last line since that isn't a new thing (the key just got renamed and RTMP removed). But otherwise, yeah, sounds good!

  [this document](https://github.com/elan-ev/opencast-studio/blob/2020-09-14/CONFIGURATION.md).

### Wowza streaming configuration changes

> In [this pull request](https://github.com/opencast/opencast/pull/1179), the configuration of Wowza servers was made

Reviewer 2: ```suggestion
In [pull request #1179](https://github.com/opencast/opencast/pull/1179), the configuration of Wowza servers was made
```

> tenant-specific. Before, you had to specify `org.opencastproject.streaming.url` in `custom.properties`. Starting with

Reviewer 2: Inline code blocks starting with `etc/` get special highlighting to explain possible locations for these files.

```suggestion
tenant-specific. Before, you had to specify `etc/org.opencastproject.streaming.url` in `custom.properties`. Starting with
```

> Opencast 9, that value has to specified as `org.opencastproject.mh_default_org.wowza.url` in
> `org.opencastproject.distribution.streaming.wowza.WowzaStreamingDistributionService.cfg`. Of course, if you have
> multiple tenants, you want to replace `mh_default_org` in that key accordingly.

Reviewer 2: ```suggestion
Opencast 9, it is now a tenant-specific configuration in `etc/org.opencastproject.distribution.streaming.wowza.WowzaStreamingDistributionService.cfg`.
```

Author: I removed the "Of course" sentence in the end, but kept my version of the first sentence as I think the key name is worth mentioning and my version avoid the repetition of "tenant specific".
